packages/compass-schema-validation/src/components/validation-states/validation-states.jsx

```diff
@@ -87,40 +89,39 @@ class ValidationStates extends Component {
     if (!this.isEditable()) {
       if (this.props.editMode.collectionTimeSeries) {
         return (
-          <StatusRow style="warning">
-            <div id="collectionTimeSeries">
-              {READ_ONLY_WARNING.collectionTimeSeries}
-            </div>
-          </StatusRow>
+          <WarningSummary
+            warnings={[READ_ONLY_WARNING.collectionTimeSeries]}
+            data-testid="collection-validation-warning"
+          />
         );
       }
 
       if (this.props.editMode.collectionReadOnly) {
         return (
-          <StatusRow style="warning">
-            <div id="collectionReadOnly">
-              {READ_ONLY_WARNING.collectionReadOnly}
-            </div>
-          </StatusRow>
+          <WarningSummary
+            warnings={[READ_ONLY_WARNING.collectionReadOnly]}
+            data-testid="collection-validation-warning"
+          />
         );
       }
 
       if (this.props.editMode.writeStateStoreReadOnly) {
         return (
-          <StatusRow style="warning">
-            <div id="writeStateStoreReadOnly">
-              {READ_ONLY_WARNING.writeStateStoreReadOnly}
-            </div>
-          </StatusRow>
+          <WarningSummary
+            warnings={[READ_ONLY_WARNING.writeStateStoreReadOnly]}
+            data-testid="collection-validation-warning"
+          />
         );
       }
 
       if (this.props.editMode.oldServerReadOnly) {
         return (
-          <StatusRow style="warning">
-            <div id="oldServerReadOnly">
-              {READ_ONLY_WARNING.oldServerReadOnly}
-              <div>&nbsp;</div>
+          <Banner
+            variant="warning"
+            warnings={[READ_ONLY_WARNING.oldServerReadOnly]}
```

Comment:
This is a leafygreen banner, right? There is no warnings prop on this component

Reply:
Ah yup! Thanks for catching.